RJL.UIP.HW3.ConditionalsMethods/Program.cs

```diff
@@ -0,0 +1,245 @@
+            Console.WriteLine("Enter value for Month");
+            string inputStringMonth = Console.ReadLine();
+            bool isSuccessMonth = int.TryParse(inputStringMonth, out inputMonthValueTask4);
+            resultCountDaysTask4 = GetCountDays(inputYearValueTask4, inputMonthValueTask4);
+            Console.WriteLine($"{inputYearValueTask4} year {inputMonthValueTask4} month contains {resultCountDaysTask4} days");
+            Console.WriteLine("-------------------------------------------------------------");
+            // task5
+            Console.WriteLine("Task5");
+            int[] randomArrayTask5 = CreateRandomArray();
+            Console.WriteLine("Random current array is");
+            PrintArray(randomArrayTask5);
+            int[] sortArrayTask5 = randomArrayTask5;
+            QuickSortArray(sortArrayTask5, 0, randomArrayTask5.Length - 1);
+            Console.WriteLine();
+            Console.WriteLine("Sorted array is ");
+            PrintArray(sortArrayTask5);
+            Console.WriteLine();
+            Console.WriteLine("-------------------------------------------------------------");
+            //task6
+            Console.WriteLine("Task6");
+            int[] randomArrayTask6 = CreateRandomArray();
+            Console.WriteLine("Random current array is");
+            PrintArray(randomArrayTask6);
+            Console.WriteLine();
+            SetNullMinMaxArray(randomArrayTask6);
+            Console.WriteLine("Array with null value between min and max is");
+            PrintArray(randomArrayTask6);
+            Console.WriteLine();
+            Console.WriteLine("-------------------------------------------------------------");
+            //task7
+            Console.WriteLine("Task7");
+            int randomArrayLengthTask7 = randomizer.Next(1, 10);
+            int[][] randomArrayTask7 = new int[randomArrayLengthTask7][];
+            Console.WriteLine("Current random array of arrays is");
+            for (int i = 0; i < randomArrayLengthTask7; i++)
+            {
+                randomArrayTask7[i] = CreateRandomArray();
+                PrintArray(randomArrayTask7[i]);
+                Console.WriteLine();
+            }
+            Console.WriteLine("Array after sorting");
+            for (int i = 0; i < randomArrayLengthTask7; i++)
+            {
+                QuickSortArray(randomArrayTask7[i], 0, randomArrayTask7[i].Length - 1);
+                PrintArray(randomArrayTask7[i]);
+                Console.WriteLine();
+            }
+            Console.WriteLine("-------------------------------------------------------------");
+            //task8
+            Console.WriteLine("Task8");
+            string[] multiplyTableArrayTask8 = GetMultiplyTable(15);
+            foreach (string item in multiplyTableArrayTask8)
+            {
+                Console.WriteLine(item);
+            }
+            Console.ReadLine();
```

Comment:
Something goes wrong (1 and 2 pair)
**_Task8
6*1=6
6*1=6
6*7=42
1*8=8
4*7=28
2*5=10
8*6=48
3*2=6
8*4=32
2*6=12
8*1=8
7*7=49
4*3=12
6*4=24
7*4=28_**